src/plugins/ui_feature/renderer.ts

```diff
@@ -733,4 +747,16 @@ export class RendererPlugin extends UIPlugin {
     }
     return boxes;
   }
+
+  /**
+   * Computes the coordinates and size to draw the zone on the canvas, but exclude the borders of the cells.
+   */
+  private getVisibleRectWithoutBorders(zone: Zone, thinLineWidth: number): Rect {
+    const visibleRect = this.getters.getVisibleRect(zone);
+    return {
+      ...visibleRect,
+      height: visibleRect.height - thinLineWidth,
+      width: visibleRect.width - thinLineWidth,
```

Comment:
Why only `- thinLineWidth` and not `- 2*thinLineWidth`?
When I think about borders, I would think they are all around. It looks like something is wrong

Also, `thinLineWidth` does not make much sense here. Why does this method needs to know it's "thin" ? Something like `borderWidth` maybe ?